src/permuter.py

```diff
@@ -118,6 +119,7 @@ def __init__(
         self._show_errors = show_errors
         self._best_only = best_only
         self._better_only = better_only
+        self.score_threshold = score_threshold
```

Comment:
make this member private, just like the other ones

Reply:
fixed, yea I had it private, but then when I did the rename, I accidentally changed it